src/main/java/src/chapter7/question3/Main.java

```diff
@@ -0,0 +1,24 @@
+package src.chapter7.question3;
+
+public class Main {
+    public static void getDivisibility(int number) {
+        boolean divisibleBy2 = number % 2 == 0;
```

Comment:
If you are not going to change this variable's value, declare it as final.

Reply:
Declared.